### ISSUE TITLE
Interface binary compatibility

### ISSUE THREAD

**Issue by vrozov** [siom79/japicmp#85]
The following seems to be binary compatible change, though it is reported as binary incompatible:

```
public interface Foo {
 void foo();
}
```
to 
```
public interface Bar {
  void foo();
}

public interface Foo extends Bar {
}
```

**Comment by siom79** [comment]
Yes, this is binary compatible but was reported to be binary incompatible.

I have just fixed this on the `develop` branch:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn install
```

**Comment by siom79** [comment]
Released with version 0.6.1.
